server/config/express.js

```diff
@@ -95,8 +95,8 @@ export default function() {
   app.use('/api', apiRoutes(API_DELAY, API_FAILURE_RATE))
 
   // Setting the static folder
-  const static_assets_path = (process.env.NODE_ENV === 'production') ? './dist/client' : './assets'
-  app.use(express.static(path.resolve(static_assets_path)))
+  if (process.env.NODE_ENV === 'production')
```

Comment:
Only need to serve static files in production, webpack dev server does it itself